packages/admin/src/api.ts

```diff
@@ -54,7 +54,10 @@ export class AdminClient extends BaseApiClient {
     return process.env.DEFENDER_ADMIN_POOL_CLIENT_ID || '40e58hbc7pktmnp9i26hh5nsav';
   }
 
-  protected getApiUrl(): string {
+  protected getApiUrl(v: ApiVersion): string {
```

Comment:
Wondering if it makes sense to make this param optional or default to either version?